src/d3/d3Renderer.js

```diff
@@ -59,6 +70,10 @@ var d3Renderer = function (arg) {
   /**
    * Meta functions for converting from geojs styles to d3.
    * @private
+   * @param {function|object} f The style value or function to convert.
+   * @param {function} [g] An optional function that returns a boolean; if it
```

Comment:
I am trying to remember..we are not using square braces for f because it could be an object?

Reply:
In jsdoc brackets around a parameter indicate that it is optional.  Here, `f` is required, but `g` is optional.

Reply:
ah.. that's it. thanks for reminding that.